.gitignore

```diff
@@ -23,6 +25,7 @@ dist
 .idea/dictionaries
 .idea/vcs.xml
 .idea/jsLibraryMappings.xml
+.idea/google-java-format.xml
```

Comment:
Is it possible to just gitignore the whole idea folder?

Reply:
That's what I would prefer, but it was added by @edalex-ian , so I won't touch it.

Reply:
I'm not too fussed. My preference is to leave it there, as we can have some project specific bits there and it saves people time. But if it keeps stressing people, then remove it and make sure we ignore it - as it was all the git warnings etc that made me add and then add the ignores.